src/main/java/com/shaneschulte/mc/clanarena/commands/CreateCmd.java

```diff
@@ -0,0 +1,379 @@
+package com.shaneschulte.mc.clanarena.commands;
+
+import com.shaneschulte.mc.clanarena.utils.CmdProperties;
+import com.shaneschulte.mc.clanarena.utils.MsgUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.Location;
+import org.bukkit.boss.BarColor;
+import org.bukkit.boss.BarStyle;
+import org.bukkit.boss.BossBar;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.Action;
+import org.bukkit.event.entity.PlayerDeathEvent;
+import org.bukkit.event.player.*;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class CreateCmd implements CmdProperties, Listener {
+
+    private static HashMap<String, BossBar> currentPlayerBossBars = new HashMap<>();
```

Comment:
In general, you should use UUID instead of player name

Reply:
okay i will, it's not needed in this context though because it resets on player disconnect